bubble_sort.rb

```diff
@@ -0,0 +1,25 @@
+def bubble_sort(array)
+  array.length.times do
+    (0...(array.length - 1)).each do |i|
+      if array[i] > array[i + 1]
+        deleted = array.delete_at(i)
+        array.insert(i + 1, deleted)
+      end
+    end
+  end
+  p array
```

Comment:
- [ ] Please take out the 'p' and just leave `array` (do this in lines 10 and 24). The reason is that you are forcing the result to be printed, even when the user may not want to print it and just use the data, for example:
![image](https://user-images.githubusercontent.com/53256139/90730979-a930c300-e28e-11ea-86a9-1e2494224cc8.png)
👉 2 printed results returned

Reply:
Yes, it was a misunderstanding but is more clear now. Thank you!